### ISSUE TITLE
Gold buttons used in mega menus have blue text

### ISSUE THREAD

**Issue by kevincrafts** [CuBoulder/tiamat-theme#1562]
The text color on gold buttons should be black, but when they are used in a mega menu, they are blue: https://www.colorado.edu/visit-campus

<img width="1434" alt="Image" src="https://github.com/user-attachments/assets/e6c00439-023c-4843-9fde-be4b587558c1" />
